src/services/message.service.ts

```diff
@@ -40,6 +41,52 @@ export class MessageService {
     }
   }
 
+  async sendReactiveMessage(
+    message: IMessage,
+    messageInfo: MessageEmbed | string,
```

Comment:
`message.send(...)` takes in data of type `APIMessageContentResolvable` would make more sense to use that

Reply:
looking at the docs I can't find anything about `APIMessageContentResolvable` further more it takes `APIMessageContentResolvable` or `MessageAdditions` which the later one takes the `MessageEmbed`

Reply:
Then, lets make a type that makes it obvious what its for
something like
```
export type IMessageSendData =
  | APIMessageContentResolvable
  | (MessageOptions & { split?: false })
  | MessageAdditions;
```